### PR TITLE
Raven

### LINKED ISSUE #19: Second Version Home Page
Upgrade Home page and make it responsive

### PR DESCRIPTION
**Pull request by Fcuervo21**
Here's the home page second version.

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -1,133 +1,280 @@
-* {
+*{
     box-sizing: border-box;
     margin: 0;
     padding: 0;
+    /* font-family: Arial, Helvetica, sans-serif; */
```
Comment (RafaGomezGlez) [inline home.css:5]: Why do you declare every font-family separated? 
And why do you declare it inside the * class?

File: home.css [hunk 2]
```diff
@@ -1,133 +1,280 @@
...
 }
-.header-nav__logo {
+.navbar{
+    width: 100%;
     display: flex;
     align-items: center;
-    justify-content: center;
-    padding-left: 50px;
-    padding-right: 50px;
+    height: 90px;
```
Comment (RafaGomezGlez) [inline home.css:29]: I noticed you used px instead of rem/vh/vw/%, why?

File: home.css [hunk 3]
```diff
@@ -1,133 +1,280 @@
...
+    color: #000;
+    font-family: 'Roboto', sans-serif;
     font-weight: bold;
 }
+nav ul li::after{
+    content: "";
+    width: 0%;
+    height: 2px;
+    background: #fca311;
+    display: block;
```
Comment (RafaGomezGlez) [inline home.css:62]: I think block is the default value for display

File: home.css [hunk 4]
```diff
@@ -1,133 +1,280 @@
...
+    background-position: center center;
+    height: 100vh;
+}
+
+.image-content-box{
+    width: 56%;
+    position: absolute;  
+    left: 50%;                        /* horizontal alignment */
+    top: 50%;                         /* vertical alignment */
+    transform: translate(-50%, -50%); /* precise centering; see link below */
```
Comment (RafaGomezGlez) [inline home.css:86]: Specify which link

File: home.css [hunk 5]
```diff
@@ -1,133 +1,280 @@
...
+}
+.main-title{
+    text-align: center;
+    font-family: 'Abel', sans-serif;
```
Comment (RafaGomezGlez) [inline home.css:90]: You should put more font-family fonts, not just one

File: home.css [hunk 6]
```diff
@@ -1,133 +1,280 @@
...
 }
-
-.header-nav__list{
-    padding: 0;
-    margin: 0;
-    flex-grow: .2;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
+.logo{
```
Comment (RafaGomezGlez) [inline home.css:31]: Class in english

File: home.css [hunk 7]
```diff
@@ -1,133 +1,280 @@
...
+}
 
-.header-nav__list__item a:hover {
-    background-color:#fca311
+.middle-section{
+    padding: 3rem;
 }
-.header-nav__list__item a:active {
-    color: red;
+.titulo-about{
```
Comment (RafaGomezGlez) [inline home.css:102]: class in english

File: home.css [hunk 8]
```diff
@@ -1,133 +1,280 @@
...
-    flex-grow: 3;
+    padding-top: 4rem;
 }
-.header-nav__theme span {
-    padding: 10px;
+.first-profile-image{
+    width: 23rem;
+    box-shadow: 10px 10px 5px grey;
+}
+.primer-parrafo{
```
Comment (RafaGomezGlez) [inline home.css:120]: class in english

File: home.css [hunk 9]
```diff
@@ -1,133 +1,280 @@
...
+    width: 40rem;
+    text-align: right;
+    font-family: 'Varela Round', sans-serif;
```
Comment (RafaGomezGlez) [inline home.css:123]: More fonts

File: home.css [hunk 10]
```diff
@@ -1,133 +1,280 @@
...
-    position: relative;
-    width: 100%;
-    max-height: auto;
+.second-middle-section{
+    padding: 3rem;
 }
-.main-background__img {
-    position: relative;
-    width: 100%;
+.about-us-segundo{
```
Comment (RafaGomezGlez) [inline home.css:130]: Class in english

File: home.css [hunk 11]
```diff
@@ -1,133 +1,280 @@
...
     display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+.segundo-parrafo{
```
Comment (RafaGomezGlez) [inline home.css:135]: Classin english

File: home.css [hunk 12]
```diff
@@ -1,133 +1,280 @@
...
+    width: 40rem;
+    text-align: left;
+    font-family: 'Varela Round', sans-serif;
```
Comment (RafaGomezGlez) [inline home.css:138]: More fonts

File: home.css [hunk 13]
```diff
@@ -1,133 +1,280 @@
...
-    width: 25%;
-    height: 345px;
-    object-fit: cover;
-    flex-grow: 1;
-}
-/*Styles for the presentation using BEM*/
-.main-presentation {
-    padding-left: 8vw;
-    padding-right: 8vw;
+.primer-tarjeta{
```
Comment (RafaGomezGlez) [inline home.css:159]: Class in english

File: home.css [hunk 14]
```diff
@@ -1,133 +1,280 @@
...
-    align-items: center;
+    padding: 0.5rem;
+    background-color: #eeeeee;
+    box-shadow: 0 6px 8px 0 rgba(0,0,0,0.7);
+    transition: 0.3s;
+    margin: 0.7rem;
 }
-.main-presentation h1 {
-    font-family: sans-serif;
+.primer-tarjeta:hover{
```
Comment (RafaGomezGlez) [inline home.css:168]: Class in english

File: home.css [hunk 15]
```diff
@@ -1,133 +1,280 @@
...
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
 }
-.main-presentation__description p{
-    text-align: justify;
-    text-justify: inter-word;
+.primer-imagen-tarjeta{
```
Comment (RafaGomezGlez) [inline home.css:171]: Class in english

File: home.css [hunk 16]
```diff
@@ -1,133 +1,280 @@
...
+    width: 20rem
 }
-.main-presentation__projects {
+.contact-section-tittle{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 2rem;
```
Comment (RafaGomezGlez) [inline home.css:178]: Simplify with padding: (padding-top, padding-right, padding-bottom, padding-left).

File: home.css [hunk 17]
```diff
@@ -1,133 +1,280 @@
...
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+    .segundo-parrafo{
+        padding-top: 2rem;
+        text-align: center;
+        width: 20rem;
+        order: 1;
+    }
```
Comment (RafaGomezGlez) [inline home.css:274]: Change the name of the classes

File: index.html [hunk 18]
```diff
@@ -1,69 +1,124 @@
...
-            <div class="header-nav__logo">
-                <svg xmlns="http://www.w3.org/2000/svg" width="35" height="35" viewBox="0 0 66.363 65.425">
-                    <path id="Icon_awesome-circle-notch" data-name="Icon awesome-circle-notch" d="M38.025,4.273V6.5a3.212,3.212,0,0,0,2.366,3.086,24.618,24.618,0,1,1-13.3,0A3.213,3.213,0,0,0,29.462,6.5V4.274a3.214,3.214,0,0,0-4.02-3.109,33.181,33.181,0,1,0,16.569-.009A3.205,3.205,0,0,0,38.025,4.273Z" transform="translate(-0.562 -1.055)"/>
-                </svg>   
+        <div class = "container">
+            <div class = "navbar">
+                <img src="https://img.icons8.com/fluent/240/000000/home.png" class="logo">
+                <nav>
+                    <ul id="menuList">
+                        <li><a href="index.html">Inicio</a></li>
```
Comment (RafaGomezGlez) [inline index.html:19]: Replace with
```suggestion
                        <li><a href="./index.html">Inicio</a></li>
```

File: index.html [hunk 19]
```diff
@@ -1,69 +1,124 @@
...
+                <img src="https://embajadoresalumni.udd.cl/files/2018/04/linkedin.png" alt="linkedin" class="imagenes-contacto">
+                <img src="https://image.flaticon.com/icons/png/512/25/25231.png" alt="github" class="imagenes-contacto">
+                <img src="https://cdn.icon-icons.com/icons2/1826/PNG/512/4202011emailgmaillogomailsocialsocialmedia-115677_115624.png" alt="correo" class="imagenes-contacto">
             </div>
         </div>
-    </main>
-    <footer>
-        <div class="footer-wrapper">
+    </div>
+    <div class="bottom-part"></div>
```
Comment (RafaGomezGlez) [inline index.html:102]: Use the footer tag, not div

File: sort/sort.html [hunk 20]
```diff
@@ -47,18 +47,16 @@
             </form>
         </div>
 
-        <canvas id="myChart"></canvas>
+        <canvas id="myChart"></canvas> </canvas>
```
Comment (RafaGomezGlez) [inline sort/sort.html:50]: It has to be
```suggestion
        <canvas id="myChart"></canvas>
```

File: sort/sort.html [hunk 21]
```diff
@@ -47,18 +47,16 @@
...
-            
-            const ctx = document.getElementById('myChart').getContext('2d');
-            const xlabels = [];
-            const myChart = new Chart(ctx, {
+            var ctx = document.getElementById('myChart');
+            var myChart = new Chart(ctx, {
                 type: 'bar',
                 data: {
-                    labels: [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15],
+                    labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
```
Comment (RafaGomezGlez) [inline sort/sort.html:56]: You can't use colors to identify each bar, when the bar changes the vlaues 'Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange' are going to be deleted.

File: sort/sort.html [hunk 22]
```diff
@@ -78,13 +76,11 @@
                         borderWidth: 1
                     }]
                 },
-            
             });
-
-        </script>
+            </script>
```
Comment (RafaGomezGlez) [inline sort/sort.html:80]: Identation

File: sort/sort.js [hunk 23]
```diff
@@ -21,17 +36,63 @@ function generateLabelsFromTable(quantity)
     return labels;
 }
 
-function updateChart() {
+function sendData() {
 
-    var sortValue = parseInt(document.getElementById("sort-method").value);
+    var sortValue = (document.getElementById("sort-method").value);
```
Comment (RafaGomezGlez) [inline sort/sort.js:41]: Why do you change it?

**Review by RafaGomezGlez — CHANGES_REQUESTED** [review]
Most of the mistakes you make are in the quality of your code, it works, but it's not clean

File: sort/sort.html (hunk 20 above)
Comment (Fcuervo21) [reply sort/sort.html:50]: That was your code

File: sort/sort.html (hunk 21 above)
Comment (Fcuervo21) [reply sort/sort.html:56]: That's was your code

File: sort/sort.js (hunk 23 above)
Comment (Fcuervo21) [reply sort/sort.js:41]: That's was your code

File: home.css (hunk 16 above)
Comment (Fcuervo21) [reply home.css:178]: That would ruin responsive design, that's the reason I used that.

**Comment by Fcuervo21** [comment]
mistakes were made choosing the wrong branch

**Pull request closed without merging**